packages/mina-loader/lib/helpers.js

```diff
@@ -1,6 +1,42 @@
 const vm = require('vm')
 const loaderUtils = require('loader-utils')
 
+/**
+ * Retrieves the public path from the loader options, context.options (webpack <4) or context._compilation (webpack 4+).
+ * context._compilation is likely to get removed in a future release, so this whole function should be removed then.
+ * See: https://github.com/peerigon/extract-loader/issues/35
```

Comment:
I was thinking maybe we could add an option (e.g. `publicPath`) to `mina-loader`, if we can't access the `context.option` or `context._compilation`.

Reply:
Although most of the time our users will keep it as `"/"`.